kv/mdbx/kv_mdbx.go

```diff
@@ -227,6 +239,15 @@ func (opts MdbxOpts) Open() (kv.RwDB, error) {
 		return nil, fmt.Errorf("%w, label: %s, trace: %s", err, opts.label.String(), stack2.Trace().String())
 	}
 
+	if opts.syncPeriod != 0 {
+		// the option value is in 1/65536 of second units
+		optValue := uint64(opts.syncPeriod / (time.Second / 65536))
```

Comment:
1. please create some public helper funcs for this conversion - because easy to make it wrong (you need multiply here, not divide - right?). 
See mdbx-go.toDuration func. Probably we need both: FromDuration/ToDuration in mdbx-go 

2. Can this option be set before env.Open?

Reply:
see also: https://github.com/erthink/libmdbx/blob/03d828834bdb57a8e0ff34a311b6b835bfe4a479/src/osal.c#L1992

Reply:
1. Agree.
2. I've tried setting it before env.Open at first, but it gave me "no permission" error. In the mdbx core.c I've found that it's caused by env not being in the "active" state. After env.Open it worked.

Reply:
PR: https://github.com/torquem-ch/mdbx-go/pull/55

Reply:
merged and created release https://github.com/torquem-ch/mdbx-go/releases/tag/v0.22.16